src/com/opc_ua/opcua_local_handler.cpp

```diff
@@ -906,9 +908,9 @@ UA_StatusCode COPC_UA_Local_Handler::initializeCreateNode(CActionInfo &paActionI
   //The main process is done in the execution
   UA_StatusCode retVal = UA_STATUSCODE_GOOD;
 
-  CSinglyLinkedList<CActionInfo::CNodePairInfo*>::Iterator it = paActionInfo.getNodePairInfo().back();
+  auto& instance = paActionInfo.getNodePairInfo().back();
```

Comment:
Maybe we should use a more explicit variable name, e.g. nodePair?

Reply:
you're right. Changed